gpt_json/gpt.py

```diff
@@ -302,19 +331,46 @@ async def submit_request(
 
         optional_parameters = {}
 
-        if max_response_tokens:
+        if max_response_tokens and self.model in [
+            GPTModelVersion.GPT_3_5,
+            GPTModelVersion.GPT_4,
+        ]:
             optional_parameters["max_tokens"] = max_response_tokens
+        elif self.model in [
+            GPTModelVersion.CLAUD,
+            GPTModelVersion.CLAUD_100K,
+        ]:
+            # need to specify a default for CLAUD models.
+            # using OpenAI's default of 16 tokens: https://platform.openai.com/docs/api-reference/completions/create#completions/create-max_tokens
```

Comment:
Is this a magic number; any empirical reason to choose 16?

Reply:
no empirical, just consistency with OAI's api